{"uid":"21f1d71370349db8","name":"test_password_changes","fullName":"tests.test_login_page.TestLoginForm#test_password_changes","historyId":"a84a9123337f0862aa26252985595c8d","time":{"start":1689186455699,"stop":1689186455699,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x56431ef8b4e3 <unknown>\n#1 0x56431ecbac76 <unknown>\n#2 0x56431ece3d78 <unknown>\n#3 0x56431ece0029 <unknown>\n#4 0x56431ed1eccc <unknown>\n#5 0x56431ed1e47f <unknown>\n#6 0x56431ed15de3 <unknown>\n#7 0x56431eceb2dd <unknown>\n#8 0x56431ecec34e <unknown>\n#9 0x56431ef4b3e4 <unknown>\n#10 0x56431ef4f3d7 <unknown>\n#11 0x56431ef59b20 <unknown>\n#12 0x56431ef50023 <unknown>\n#13 0x56431ef1e1aa <unknown>\n#14 0x56431ef746b8 <unknown>\n#15 0x56431ef74847 <unknown>\n#16 0x56431ef84243 <unknown>\n#17 0x7f387e627b43 <unknown>","statusTrace":"E   selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n      (unknown error: DevToolsActivePort file doesn't exist)\n      (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n    Stacktrace:\n    #0 0x56431ef8b4e3 <unknown>\n    #1 0x56431ecbac76 <unknown>\n    #2 0x56431ece3d78 <unknown>\n    #3 0x56431ece0029 <unknown>\n    #4 0x56431ed1eccc <unknown>\n    #5 0x56431ed1e47f <unknown>\n    #6 0x56431ed15de3 <unknown>\n    #7 0x56431eceb2dd <unknown>\n    #8 0x56431ecec34e <unknown>\n    #9 0x56431ef4b3e4 <unknown>\n    #10 0x56431ef4f3d7 <unknown>\n    #11 0x56431ef59b20 <unknown>\n    #12 0x56431ef50023 <unknown>\n    #13 0x56431ef1e1aa <unknown>\n    #14 0x56431ef746b8 <unknown>\n    #15 0x56431ef74847 <unknown>\n    #16 0x56431ef84243 <unknown>\n    #17 0x7f387e627b43 <unknown>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"browser","time":{"start":1689186455700,"stop":1689186456354,"duration":654},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x56431ef8b4e3 <unknown>\n#1 0x56431ecbac76 <unknown>\n#2 0x56431ece3d78 <unknown>\n#3 0x56431ece0029 <unknown>\n#4 0x56431ed1eccc <unknown>\n#5 0x56431ed1e47f <unknown>\n#6 0x56431ed15de3 <unknown>\n#7 0x56431eceb2dd <unknown>\n#8 0x56431ecec34e <unknown>\n#9 0x56431ef4b3e4 <unknown>\n#10 0x56431ef4f3d7 <unknown>\n#11 0x56431ef59b20 <unknown>\n#12 0x56431ef50023 <unknown>\n#13 0x56431ef1e1aa <unknown>\n#14 0x56431ef746b8 <unknown>\n#15 0x56431ef74847 <unknown>\n#16 0x56431ef84243 <unknown>\n#17 0x7f387e627b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/testing_foodgram_project_react/testing_foodgram_project_react/conftest.py\", line 26, in browser\n    browser = webdriver.Chrome(\n              ^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 84, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 104, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"feature","value":"Тесты аутентификации зарегистрированного пользователя."},{"name":"story","value":"Тест изменения пароля."},{"name":"tag","value":"password_changes_test(scope='function')"},{"name":"tag","value":"login_form_test(scope='class')"},{"name":"tag","value":"run(order=2)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login_page"},{"name":"subSuite","value":"TestLoginForm"},{"name":"host","value":"fv-az256-210"},{"name":"thread","value":"2518-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["login_form_test(scope='class')","run(order=2)","password_changes_test(scope='function')"]},"source":"21f1d71370349db8.json","parameterValues":[]}